process/economics/economicsData.go

```diff
@@ -291,14 +304,14 @@ func (ed *EconomicsData) NumNodes() uint32 {
 	return ed.numNodes
 }
 
-// AuctionEnabled returns whether full auction process is enabled
-func (ed *EconomicsData) AuctionEnabled() bool {
-	return ed.auctionEnabled
+// AuctionEnableNonce returns whether full auction process is enabled
+func (ed *EconomicsData) AuctionEnableNonce() uint64 {
+	return ed.auctionEnableNonce
 }
 
-// SetAuctionEnabled sets whether the auction is enabled
-func (ed *EconomicsData) SetAuctionEnabled(auctionEnabled bool) {
-	ed.auctionEnabled = auctionEnabled
+// StakeEnableNonce returns whether full auction process is enabled
```

Comment:
`// StakeEnableNonce returns the nonce that will trigger staking beginning`

Reply:
done.